sidebars.js

```diff
@@ -9,6 +9,14 @@ module.exports = {
       "tutorial/intro",
       "tutorial/basic"
     ],
-    "API Reference": [],
+    "API Reference": [
+      "api/core/bind",
+      "api/core/share-latest",
+      {
+        "type": "category",
+        "label": "Utils",
+        "items": []
+      }
+    ],
```

Comment:
My idea here is to have `@react-rxjs/core` docs in the root API Reference, and utils in a sub-category.

Reply:
I actually like how recoil has done it, with core and utils explicitly categorised to match the npm libs, but this works too.